Assets/Example/Scenes/EntryPoint.unity

```diff
@@ -231,7 +231,7 @@ RectTransform:
   m_Children:
   - {fileID: 1923053171}
   m_Father: {fileID: 2006651435}
-  m_RootOrder: 19
```

Comment:
Why does this scene file get changed?

Reply:
Ui added for testing invalid segmentation.

Reply:
Share a screenshot of that UI

Reply:
still not uploaded